isis/python_bindings/UserInterface.i

```diff
@@ -0,0 +1,54 @@
+%module(package="astroset") UserInterface
+
+
+%{
+  #include "Pvl.h"
```

Comment:
It doesn't seem like this is really being used anywhere? I could totally be wrong though. If it's not we may want to avoid including it w.r.t. the pvl_core stuff

Reply:
I think it was a way to just use PVL when messing with some of the interface stuff. i.e. dumping the env to a PVL object. Didn't use it in the actual script